es6.js

```diff
@@ -0,0 +1,29 @@
+'use strict';
+module.exports = {
+  extends: ['eslint-config-mm/es5'],
+  parserOptions: {
+    ecmaVersion: 6
+    // Can't use ES6 imports yet in node: https://github.com/nodejs/help/issues/53#issuecomment-165763446
+    // sourceType: 'module'
+  },
+  env: {
+    es6: true
+  },
+  rules: {
+    'array-callback-return': 1,
+    'arrow-parens': [1, 'always'],
+    'arrow-body-style': [1, 'always'],
+    'arrow-spacing': 1,
+    'no-const-assign': 1,
```

Comment:
I miss the rule `no-confusing-arrows` (http://eslint.org/docs/rules/no-confusing-arrow), which is pointed out in the AirBNB styleguide here: https://github.com/airbnb/javascript#arrows--confusing. Did you intentionally leave it out?